pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	automationConfigKey   = "automation-config"
+	AutomationConfigKey   = "automation-config"
 	agentName             = "mongodb-agent"
 	mongodbName           = "mongod"
 	agentImageEnvVariable = "AGENT_IMAGE"
```

Comment:
we need to be consistent: either all capitalised or none, no?

![](https://media.giphy.com/media/NEJCPYzr6xmAU/giphy.gif)

Reply:
this one in particular is being imported in the e2e test.